omnibus/config/software/datadog-agent-integrations-py3.rb

```diff
@@ -225,29 +232,26 @@
     end
 
     File.open("#{static_reqs_in_file}", 'r+').readlines().each do |line|
-      blacklist_flag = false
-      blacklist_packages.each do |blacklist_regex|
-        re = Regexp.new(blacklist_regex).freeze
+      excluded_packages.each do |package_regex|
+        re = Regexp.new(package_regex).freeze
         if re.match line
-          blacklist_flag = true
+          next
```

Comment:
I'm not sure this works as intended because of the nested blocks. I think this `next` would just skip an iteration on the `excluded_packages.each do |package_regex|` block, and that's not what the original code does.

Reply:
You may consider doing something like this to replace that block:
```ruby
  next if excluded_packages.any? { |package_regex| line.match(package_regex) }
```

Reply:
Indeed, good catch! Updated with your proposal